### PR TITLE
Use applied directives rather than directives by themselves

### PR DESCRIPTION
**Pull request by pelletier197**
Update to use the new `AppliedDirectives` instead of the old `Directive`. 

This seems to break when upgrading to graphql-java 19, and it does still work for graphql-java 18.

### CODE REVIEW

**Review by enriquedacostacambio — APPROVED** [review]

**Comment by pelletier197** [comment]
@enriquedacostacambio any chance we can get this merged if it's okay? This is stopping us from updating to the latest GraphQL tools. 

Thanks in advance!

File: src/main/java/graphql/validation/interpolation/ResourceBundleMessageInterpolator.java [hunk 1]
```diff
@@ -27,12 +16,15 @@
...
-import static java.lang.annotation.ElementType.METHOD;
-import static java.lang.annotation.ElementType.PARAMETER;
-import static java.lang.annotation.ElementType.TYPE_USE;
+
+import jakarta.validation.Constraint;
+import jakarta.validation.Path;
+import jakarta.validation.Payload;
+import java.lang.annotation.Retention;
+import java.lang.annotation.Target;
+import java.util.*;
```
Comment (bbakerman) [inline src/main/java/graphql/validation/interpolation/ResourceBundleMessageInterpolator.java:25]: This annoys me - please turn on expand all imports

**Review by bbakerman — CHANGES_REQUESTED** [review]
I am happy to accept this PR howev er I would really like it to not use `.*` imports on the changed files.

Could you please go back and re-format this please

**Comment by bbakerman** [comment]
Ok progress over perfection - I will let this in and we can fix up the `.*` imports later - could I ask  that another PR be raised for that please @pelletier197

**Comment by pelletier197** [comment]
> Ok progress over perfection - I will let this in and we can fix up the `.*` imports later - could I ask that another PR be raised for that please @pelletier197

Yeah no problem, I'll do that today. Sorry about that, it's the default setting of Intellij and I didn't see it.

**Pull request merged**